src/check_oldies/commands.py

```diff
@@ -1,5 +1,9 @@
 import os
-import pipes
+# FIXME (dbaty, 2024-03-28): `pipes` is deprecated since Python 3.11,
+# will be removed in 3.13. We should use `subprocess` instead (but
+# we'll have to make sure that we can do that and still support 3.8
+# and 3.9). or perhaps get rid of it?
```

Comment:
3.8 and 3.9 should not be supported as they are near the end of their security cycle and close to end-of-life : https://devguide.python.org/versions/

Reply:
What are potential limitations regarding 3.8 and 3.9 ?

Reply:
I am not inclined to remove support of versions that have not yet reached their end of life (however old they might be).